addons/website_slides/models/slide_channel.py

```diff
@@ -229,10 +280,19 @@ def _get_default_enroll_msg(self):
     partner_ids = fields.Many2many(
         'res.partner', 'slide_channel_partner', 'channel_id', 'partner_id',
         string='Members', help="All members of the channel.", context={'active_test': False}, copy=False, depends=['channel_partner_ids'])
-    members_count = fields.Integer('Attendees count', compute='_compute_members_count')
-    members_done_count = fields.Integer('Attendees Done Count', compute='_compute_members_done_count')
+    members_count = fields.Integer('Attendees count', compute='_compute_members_counts')
+    members_invitation_pending_count = fields.Integer('# Attendees Pending', compute='_compute_members_counts')
+    members_active_count = fields.Integer(
```

Comment:
Ok I get it, partners are the res.partner records of the corresponding members channel_partner_ids.
And we would introduce two variables, partner_ids and partner_all_ids on the model. Same applies to wording of these variables. Here member_count would mean the enrolled member count, since no all is added. Seems fair.